wrappers/nodejs/test/test-functional-online.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2017 Intel Corporation. All rights reserved.
+// Use of this source code is governed by an Apache 2.0 license
+// that can be found in the LICENSE file.
+
+'use strict';
+
+/* global describe, it, before, after */
+const assert = require('assert');
+const fs = require('fs');
+let rs2;
+try {
+  rs2 = require('node-librealsense');
+} catch (e) {
+  rs2 = require('../index.js');
+}
+
+let isRecord;
+let isPlayback;
+let recordFile;
+
+for (let i = 0; i < process.argv.length; i++) {
+  if (process.argv[i] === '--record') {
+    isRecord = true;
+    recordFile = process.argv[i + 1];
+  } else if (process.argv[i] === '--playback') {
+    isPlayback = true;
+    recordFile = process.argv[i + 1];
+  }
+}
```

Comment:
I would suggest using existing cmd arguments parse module like https://www.npmjs.com/search?q=optparse&page=1&ranking=optimal .

Reply:
Yes, and after checking the logic, the playback checking code was actually not used, so I will remove them.